corehq/apps/accounting/filters.py

```diff
@@ -237,7 +237,10 @@ def filter_context(self):
 
     @classmethod
     def get_date_str(cls, request, date_type):
-        return request.GET.get('%s_%s' % (cls.slug, date_type))
+        date_str = request.GET.get('%s_%s' % (cls.slug, date_type))
+        if date_str is None:
+            date_str = request.POST.get('%s_%s' % (cls.slug, date_type))
+        return date_str
```

Comment:
Was this an existing bug that was found, or was there a chance in this PR that resulted in a need for checking POST requests as well?

Reply:
I can't remember exactly which report it was (probably `CaseActivity` and `ApplicationStatusReport` which don't have date ranges so are handled differently in a few places) but yep, I noticed that for some reason they didn't have a `date_str` in their `GET` but one in their `POST`.